### ISSUE TITLE
Attachment on the Walkthrough and Enhanced walkthrough videos

### ISSUE THREAD

**Issue by sean8300** [sean8300/listerassister_issues#715]
Would there be a way for the email we generate through the dashboard to have an attachment or link to a document that Zillow gives us for both the Walkthrough and Advanced WT emails? 
 Submitted By:  Jessica Owen

**Comment by Juvie92** [comment]
Doesn't seem attachments are possible but links are.
I was hoping to simply add a link to this on Zillow's website but they do not seem to have it anywhere.
I'm thinking we can upload it to listerassister.info and link to it that way (like the shot list).
Will have to wait on that so putting this on hold.

**Comment by Juvie92** [comment]
Decided to upload to dropbox and link directly to the file. When clicked file opens right up. There are a few dropbox menus that get annoying but nothing too bad.

Walkthrough and Enhanced WT templates updated on live with link.
